ash/src/extensions/ext/descriptor_indexing.rs

```diff
@@ -0,0 +1,10 @@
+use std::ffi::CStr;
+
+#[derive(Clone)]

```

Comment:
I don't think we have an extension in ash yet that doesn't contain any function pointers. I guess it is more consistent. @Ralith Any opinions? I am generally okay with "empty" extensions.

Reply:
Seems fine to me; as you say, it's nice for consistency. There are many empty `*Fn` already, anyway.

Reply:
Sounds great